### ISSUE TITLE
Names for test classes are inconsistent

### ISSUE THREAD

**Issue by nickssl** [spedas/pyspedas#1005]
This is a minor issue which is easy to fix.

VSCode has a separate section for tests and it lists all the tests in the project (see attached image). This is very useful for debugging. This feature would be easier to use if the names for the Class objects in the various tests.py files follow a consistent naming scheme. One possibility is to name all the classes "LoadTestCases" since the majority of the existing tests already use this name. Another possibility is to use "folder-name_tests", for example Kompsat_Tests, Fast_Tests, and so on.  This is an easy change to make, and as far as I know, it does not affect anything else. 

![tests](https://github.com/user-attachments/assets/fe03b6b3-0477-44b3-bdde-15e7af464077)

**Comment by jameswilburlewis** [comment]
Yes, it would probably be good to make these more descriptive.  I know I'm frequently guilty of just cloning a test.py file and replacing the test methods....at the very least, the class names should include the project name or category the tests are meant to be exercising.
